en/docs/references/ballerina-expression-syntax.md

```diff
@@ -0,0 +1,421 @@
+# Ballerina Expression Syntax
+
+Ballerina expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you with the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (`"`)  to mark the boundary between the starting and ending characters.
+
+#### Examples
```

Comment:
Why we need examples as headings? It clutters the left navigation . See https://github.com/wso2-enterprise/choreo/issues/3824

Reply:
Yes, examples in this case examples can be a heading. Ideally the left navigation should only have the level 2 and level 3 headings included in the "on this page" navigation. It doesn't make sense to show more than that because it is not readable.

Reply:
No one wants to refer examples via links no? Even if the console wants to refer to a section we can link to the corresponding sub-topic not the examples. So instead of making examples sub-topic can't we use different formatting? Like bold text etc?

Reply:
Using bold text, will result in text in the same font size be bold. It won't honor the heading levels, and styling. Refer to the image attached.  I am not quite sure if you can add a custom paragraph style to a section of text. 
![Screenshot from 2021-05-12 10-12-47](https://user-images.githubusercontent.com/7173347/117919454-a6896880-b30a-11eb-84fc-87b409271411.png)